CMakeLists.txt

```diff
@@ -756,6 +778,16 @@ if(APPLE AND CMAKE_GENERATOR MATCHES "Xcode")
   set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} ${US_LINK_FLAGS}")
 endif()
 
+#-----------------------------------------------------------------------------
+# Deterministic builds environment variables
+#-----------------------------------------------------------------------------
+if(US_USE_DETERMINISTIC_BUNDLE_BUILDS)
+  # On windows, setlocale() is used to set these variables in the ResourceCompiler
```

Comment:
an explanation of why the locale matter for deterministic bundles would be helpful.